signal-slot-benchmarks/vs/signal-slot-benchmarks.vcxproj

```diff
@@ -132,7 +132,7 @@
       <EnableModules>false</EnableModules>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <OmitFramePointers>false</OmitFramePointers>
-      <AdditionalIncludeDirectories>C:\boost_1_71_0;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\boost_1_71_0;C:\Projects\signal-slot-benchmarks\benchmark\lib\iscool\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
local path in project file

Reply:
Same here, I have copied upstream's file. I will push a fixed version but I can't test it.